Example/Authenticator/View Controllers/Connection/ConnectViewController.swift

```diff
@@ -95,3 +90,18 @@ extension ConnectViewController: CompleteViewDelegate {
         cancelPressed()
     }
 }
+
+// MARK: - ConnectViewModelEventsDelegate
+extension ConnectViewController: ConnectViewModelEventsDelegate {
+    func showNoInternetConnectionAlert() {
+        self.showInfoAlert(
```

Comment:
Remove `self.`.

`self.` is needed only when you need to call something inside a closure, like in `completion` block below